### PR TITLE
(maint) Add useful error message for the rake tasks that have been moved

### PR DESCRIPTION
**Pull request by shrug**

### CODE REVIEW

**Comment by underscorgan** [comment]
👍

File: tasks/deprecated.rake [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Emit useful messages for tasks that have been moved or removed.
+
+
+deprecation_message_winston = "The ticket generation tasks have moved. These now live in Winston: https://github.com/puppetlabs/winston"
+
+task :new_server_platform_tickets do
```
Comment (melissa) [inline tasks/deprecated.rake]: do these need to be namespaced?

File: tasks/deprecated.rake [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  warn deprecation_message_winston
+end
+
+task :platform_addition do
```
Comment (melissa) [inline tasks/deprecated.rake]: This was previously `pl:agent_tickets`

File: tasks/deprecated.rake (hunk 1 above)
Comment (shrug) [inline tasks/deprecated.rake]: I'm not sure, were they before?

File: tasks/deprecated.rake (hunk 2 above)
Comment (shrug) [inline tasks/deprecated.rake]: do you know if the others were?

Comment (melissa) [inline tasks/deprecated.rake]: I checked the others. This was the only one that was different. However, they were all namespaced under `pl`

https://github.com/puppetlabs/packaging/pull/563/files

Comment (shrug) [inline tasks/deprecated.rake]: Thanks @melissa, does it look sane now?

File: tasks/deprecated.rake [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Emit useful messages for tasks that have been moved or removed.
+
+
+deprecation_message_winston = "The ticket generation tasks have moved. These now live in Winston: https://github.com/puppetlabs/winston"
+
+namespace :pl do
+  desc "The ticket generation tasks have moved. These now live in Winston: https://github.com/puppetlabs/winston"  
+  task :new_server_platform_tickets do
```
Comment (melissa) [inline tasks/deprecated.rake]: Was this task ever in packaging? I'm not seeing it in https://github.com/puppetlabs/packaging/pull/563/files but it's possible I'm missing something

Comment (shrug) [inline tasks/deprecated.rake]: I'm not sure if it was ever renamed or there before the move, I figure it doesn't hurt.

**Comment by underscorgan** [comment]
aww aww rubocop :(

**Comment by melissa** [comment]
WHITESPACE!!!!

**Comment by underscorgan** [comment]
if you get rid of the whitespace this looks 👍

**Comment by shrug** [comment]
bad copypasta! bad!

**Pull request merged**